modules/tinymce/src/models/dom/main/ts/table/api/TableResizeHandler.ts

```diff
@@ -120,14 +121,53 @@ export const TableResizeHandler = (editor: Editor): TableResizeHandler => {
   const destroy = () => {
     tableResize.on((sz) => {
       sz.destroy();
+      tableResize.clear();
     });
 
     resizeWire.on((w) => {
       TableWire.remove(editor, w);
+      resizeWire.clear();
     });
   };
 
-  editor.on('init', () => {
+  const bindEvents = (sz: TableResize) => {
+    sz.events.startDrag.bind((_event) => {
+      isDragging.set(true);
+      selectionRng.set(editor.selection.getRng());
+      editor.dispatch('blur', { focusedEditor: null });
```

Comment:
I don't think this is correct interacting with editor UI should not dispatch a blur event of the editor. Blur should only happen if you move focus out side the editor content or editor UI. The resize bars are part of the editor content UI. Oddly enough it does trigger blur/focus on inline today but not in iframe mode but I think that is a bug and I think we should continue on that path. Both iframe and inline mode should work the same here ideally.